x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportOpenJobAction.java

```diff
@@ -593,10 +590,9 @@ public void onTimeout(TimeValue timeout) {
     }
 
     private void clearJobFinishedTime(String jobId, ActionListener<OpenJobAction.Response> listener) {
-        clusterService.submitStateUpdateTask("clearing_job_finish_time [" + jobId + "]", new ClusterStateUpdateTask() {
+        clusterService.submitStateUpdateTask("clearing-job-finish-time [" + jobId + "]", new ClusterStateUpdateTask() {
```

Comment:
sorry for insisting on that but so that task names are consistent (should make it easier to search through them) could you change to `"clearing-finish-time-for-job-" + jobId`?